Build broken for OpenSUSE tumbleweed

https://github.com/phkaeser/wlmaker/actions/runs/15389052136/job/43294088228?pr=266

Contains wlroots 0.19:

```
(277/290) Installing: libwlroots-0_19-0.19.0-1.1.x86_64 [..done]
```

```
-- Checking for module 'wlroots-0.18>=0.18'
--   Package 'wlroots-0.18' not found
[...]
CMake Error at src/CMakeLists.txt:120 (TARGET_LINK_LIBRARIES):
  Target "wlmaker_lib" links to:

    PkgConfig::WLROOTS

  but the target was not found.  Possible reasons include:

    * There is a typo in the target name.
    * A find_package call is missing for an IMPORTED target.
    * An ALIAS target is missing.
[...]
```

--- a/.github/workflows/build-for-opensuse-tumbleweed.yml
+++ b/.github/workflows/build-for-opensuse-tumbleweed.yml
@@ -2,9 +2,9 @@ name: Build for openSUSE Tumbleweed
 
 on:
   push:
-    branches: [ "disabled-main" ]
+    branches: [ "main" ]
   pull_request:
-    branches: [ "disabled-main" ]
+    branches: [ "main" ]
 
 jobs:
   test:

